packages/create-plugin/templates/common/playwright.config

```diff
@@ -0,0 +1,52 @@
+import { defineConfig, devices } from '@playwright/test';
+import { dirname } from 'node:path';
+
```

Comment:
```suggestion
import type { PluginOptions } from '@grafana/plugin-e2e';

```

Reply:
Thanks for spotting this - should be done now. 👍